docs/manager/graphql-reference/schema.graphql

```diff
@@ -1907,6 +1907,12 @@ type Mutations {
   forget_image_by_id(image_id: String!): ForgetImageById
   forget_image(architecture: String = "x86_64", reference: String!): ForgetImage
 
+  """Added in 25.3.1"""
+  purge_image_by_id(image_id: String!): PurgeImageById
+
+  """Added in 25.3.1"""
+  purge_image(architecture: String = "x86_64", reference: String!): PurgeImage
```

Comment:
What's the difference between purge_image_by_id?

Reply:
`purge_image` is an API that deletes an image using its name as a string,
whereas `purge_image_by_id` directly deletes an image using its `image_id` (DB ID).

Reply:
It seems like it's nice to gradually deprecate the name-based APIs.